shell/store/prefs.js

```diff
@@ -4,6 +4,7 @@ import { clone } from '@shell/utils/object';
 import { SETTING } from '@shell/config/settings';
 
 const definitions = {};
+let prefsBeforeLogin = {};
```

Comment:
Please do not forget to provide documentation for your peers about added logic.

Reply:
Could you articulate a bit more the logic of when this happen? Because the values are not stored on login, rather in a second request. E.g.:

```ts
/**
 * Values are stored before login only here and cookies due lack of access permission. 
 * These values will be updated with a further request
 */
```

Reply:
Could you articulate a bit more the logic of when this happen? Because the values are not stored on login, rather in a second request. E.g.:

```ts
/**
 * Values are stored before login only here and cookies due lack of access permission. 
 * These values will be updated with a further request
 */
```

Reply:
Done